_data/projects/ReactWeatherApp.yml

```
@@ -0,0 +1,11 @@
+name: React Weather Application
+desc: This is a very simple React Js based weather application, that uses Weather API. 
+site: https://stellar-pie-6964b6.netlify.app/
+tags:
+  - reactjs
+  - javascript
+  - hacktoberfest
+  - open-source
+goodfirstissue:
+  name : good first issue
```

:art: almost there - this needs to be named explicitly to match what the CI error warns about:

```suggestion
upforgrabs:
  name: good first issue
```